nautobot/core/templates/admin/change_list.html

```diff
@@ -0,0 +1,133 @@
+{% load i18n admin_urls static admin_list bootstrapped_goodies_tags %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "admin/css/changelists.css" %}" />
+  {% if cl.formset %}
+    <link rel="stylesheet" type="text/css" href="{% static "admin/css/forms.css" %}" />
+  {% endif %}
+  {% if cl.formset or action_form %}
+    {% url 'admin:jsi18n' as jsi18nurl %}
+    <script type="text/javascript" src="{{ jsi18nurl|default:'../../jsi18n/' }}"></script>
+  {% endif %}
+  {{ media.css }}
+{% endblock %}
+
+{% block extrahead %}
+{{ block.super }}
+{{ media.js }}
+{% if action_form %}{% if actions_on_top or actions_on_bottom %}
+<script type="text/javascript">
+(function($) {
+    $(document).ready(function($) {
+        $("tr input.action-select").actions();
+    });
+})($);
+</script>
+{% endif %}{% endif %}
+{% endblock %}
+
+{% if not is_popup %}
+    {% block breadcrumbs %}
+        <div class="{% if cl.search_fields %}col-sm-8 col-md-9{% else %}col-sm-12 col-md-12{% endif %}">
+            <ul class="breadcrumb">
+                <li><a href="{% url 'admin:index' %}">{% trans 'Admin Home' %}</a></li>
+                <!--<li><a href="{% url 'admin:app_list' app_label=cl.opts.app_label %}">{% render_with_template_if_exist app_label|lower|add:"/admin_app_name.html" app_label %}</a></li>-->
+                <li><a href="{% url 'admin:app_list' app_label=cl.opts.app_label %}">{% with app_label=cl.opts.app_config.verbose_name|default:cl.opts.app_label %}{{ app_label|title }}{% endwith %}</a></li>
+                <li>{{ cl.opts.verbose_name_plural|capfirst }}</li>
+            </ul>
+        </div>
+        <div class="col-sm-4 col-md-3">
+            {% block search %}{% search_form cl %}{% endblock %}
+        </div>
+    {% endblock %}
+{% endif %}
+
+{% block object-tools %}
+<div id="content-navbar-collapse" class="navbar-collapse collapse">
+    {% block object-tools-items %}
+        {% if has_add_permission %}
+                {% url cl.opts|admin_urlname:'add' as add_url %}
+                <a role="button" href="{% add_preserved_filters add_url is_popup %}" class="btn btn-primary">
+                    <span class="glyphicon glyphicon-plus"></span> {% blocktrans with cl.opts.verbose_name as name %}Add{% endblocktrans %}
+                </a>
+            </li>
+        {% endif %}
+    {% endblock %}
+
+    {% comment %} {% if cl.has_filters %}
+        <ul class="nav navbar-nav navbar-right">
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown">{% trans 'Filter' %} <span class="caret"></span></a>
+                <ul class="dropdown-menu pull-right scrollable-dropdown-menu">
+                    {% for spec in cl.filter_specs %}
+                        {% admin_list_filter cl spec %}
+                    {% endfor %}
+                </ul>
+            </li>
+        </ul>
+    {% endif %} {% endcomment %}
```

Comment:
Intentional that this block is commented out? Is there something here that needs to be fixed, or should this just be removed?